web_client/.eslintrc.js

```diff
@@ -5,7 +5,7 @@ module.exports = {
   },
   extends: [
     'plugin:vue/recommended',
-    'plugin:vuetify/recommended',
+    // 'plugin:vuetify/recommended',
```

Comment:
Can you explain this change?

Reply:
This is temporary. Vuetify was causing the linting to break b/c it was throwing errors and modifying the code to comply with Vuetify's recommendations would cause significant layout shift. I've made the changes on a branch I've got but I think we should usher the changes in more slowly (and with some more tests) to ensure we understand the impact.